### DISCUSSION TITLE
Deploy Next.js with typescript

### DISCUSSION THREAD

**Discussion by FemtDeveloper** [vercel/community#684] [Help] [answered]
I got a problem deploying my shop in vercel, in Heroku and locally works fine but only in vercel doesn't. It shows me the client-fetch-error but i have the NEATAUTH_URL setted as an env. variable in Vercel. I think that my error is when i use getProviders because when i commented for deploying the page rendered ok.

this are the logs in vercel : 

![image](https://user-images.githubusercontent.com/7281791/175874354-218bea3d-f88a-44e2-b2bd-041ed8a0e8ce.png)

and this are the logs in chrome:

![image](https://user-images.githubusercontent.com/7281791/175874389-af835b63-ff48-4824-89cf-4c943398fe07.png)

this is my [nextauth].ts

![image](https://user-images.githubusercontent.com/7281791/175873901-1781efeb-b46b-4646-ae91-896c382895c8.png)

and this is my `pages/auth/login.tsx`

![image](https://user-images.githubusercontent.com/7281791/175874244-bfbfd506-e9a3-4eff-be9f-62ca79298aca.png)

could you help me please?

**Comment by swarnava** [comment, marked as answer]
Hi there,

This doesn't look like Vercel specific issue but rather on code. Can you try reaching out to Next-auth community? https://github.com/nextauthjs/next-auth/discussions
